GVRf/Framework/jni/engine/importer/assimp_importer.cpp

```diff
@@ -68,4 +71,176 @@ std::shared_ptr<Mesh> AssimpImporter::getMesh(int index) {
 
     return std::shared_ptr < Mesh > (mesh);
 }
+
+void AssimpImporter::sceneRecursion(aiNode* node, const aiScene* aScene, std::shared_ptr<Scene> scnPtr, JNIEnv * env, jobject obj, jobject bitmap, aiMatrix4x4 accumulatedTransform)
+{
+    for(int i=0; i < node->mNumMeshes; i++)
+    {
+        std::shared_ptr<SceneObject> scnObject(new SceneObject());
+
+        // Creates a scene object for the node children meshes.
+        if(scnObject->render_data() == NULL)
```

Comment:
Is there any case that render_data() is not null, then making a comment in the source here would be helpful?